modules/plymouth/nixos.nix

```diff
@@ -41,6 +42,40 @@ let
     " > $themeDir/stylix.plymouth
   '';
 
+  mkPlymouthFont =
+    font:
+    pkgs.runCommand "${font.package.name}.otf"
+      { FONTCONFIG_FILE = pkgs.makeFontsConf { fontDirectories = [ font.package ]; }; }
+      ''
+        if ! matchingFonts="$(
+          # sort because otherwise fc-match doesn't print all available formats
+          ${lib.getExe' pkgs.fontconfig "fc-match"} \
+            --sort \
+            --format '%{family}|%{file}\n' \
+            '${font.name}' | \
+            # Filter for fonts that match the name exactly and only print the path
+            awk --field-separator '|' '/^${font.name}\|/ { print $2; found=1} END {if (!found) exit 1}'
+        )"; then
+          printf 'Font not found: %s' '${font.name}' >&2
+          exit 1
+        fi
+        echo "fonts: '$matchingFonts'" >&2
```

Comment:
Would this not be `echo`ed on every build? Should we remove this debugging statement:

```suggestion
```